.github/workflows/build.yml

```diff
@@ -4,8 +4,14 @@ on:
     branches:
       - 'main'
       - 'release/*'
+    paths-ignore:
+      - 'proposals/**'
```

Comment:
```suggestion
      - 'documentation/**'
```

Reply:
You will, of course, ask, *how* on Earth was I able to make this mistake